doc/asciidoc/pagerank.adoc

```diff
@@ -118,97 +120,100 @@ As we might expect, the Home page has the highest PageRank because it has incomi
 We can also see that it's not only the number of incoming links that is important, but also the importance of the pages behind those links.
 // end::stream-sample-graph-explanation[]
 
-== Example Usage
+== Example usage
```

Comment:
The header-casing in the changes is not consistent, which style do we want to go with?